orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -483,6 +488,7 @@ func TestBaseOrchestratorMconfigBuilder_Build(t *testing.T) {
 				},
 				DynamicServices: nil,
 				FeatureFlags:    nil,
+				Orc8RVersion:    "0.0.-1",
```

Comment:
Looks like all these tests show the version not being set. Let's add 1 test where we set the version and observe its value in the mconfig

Note that this will potentially be easier given my comments about passing the orc8r_version in as an input rather than always reading it as an env variable